docs/parameters/ms.parameters.md

```diff
@@ -42,6 +42,8 @@ will be carried as strings.
 
 The following types of parameters are designed: 
 
+- **primitive**: Primitive datatype. for e.g. for parameters like {"id":114027}, {"id":true}, {"id":93.3453}
```

Comment:
add  "PRIMITIVE type is for those cases where the value should not be quoted in the final request. While LIST type is for those cases where the value should be quoted."

Reply:
done